scripts/performance/common.py

```diff
@@ -87,6 +87,28 @@ def get_artifacts_directory() -> str:
     '''
     return os.path.join(get_repo_root_path(), 'artifacts')
 
+def rename_upload_files(files) -> None:
+    for file in files:
+        directory_name = os.path.dirname(file)
+        filename = os.path.basename(file)
+        newname = "{0}-{1}".format(os.path.join(
+                                    directory_name,
+                                    os.getenv('HELIX_WORKITEM_ID')),
```

Comment:
nit: can you make passing in HELIX_WORKITEM_ID a param since this is now a common function?

Reply:
Sounds good.